internal/driver/common/canvas.go

```diff
@@ -515,6 +509,8 @@ type RenderCacheNode struct {
 	// it should free all associated resources when released
 	// i.e. it should not simply be a texture reference integer
 	painterData interface{}
+
+	needUpdateLayout bool
```

Comment:
Does this need to be within the struct for recursion? It looks to me like it only is needed within the function above and could be a local variable instead, or am I missing something?

Reply:
Indeed, good idea.